### DISCUSSION TITLE
R 5.0.1 Sunbird ED - PORTAL/DESKTOP APP - Circle QA Sign Off

### DISCUSSION THREAD

**Discussion by manju4201** [Sunbird-Ed/Community#195] [General Discussion]
Hi All,

This is Consolidated circle QA Sign off for the release 5.0.1 for the Sunbird ED Building block.

Environment: Sunbird ED Staging
Testing Status: Completed.
Defects Dashboard: NA

Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1pP1eRNXYRNTiJe0BYZ2BowHRJ9ygCi6VKIRSJ2cgFMM/edit#gid=1581973510)

Scope: [Link](https://project-sunbird.atlassian.net/issues/?filter=12641)

Circle Sign-Off Template: [Link](https://docs.google.com/spreadsheets/d/1jail-mLcIooaYbRIIRWVeFmXfw90oNoY_g_SYayp2Wg/edit#gid=0)

CC : @luckynvdu  @sharathkashyap @rajeevsathish @chitranshu-keshav @ShailandraOjha @anunagaraj

**Comment by anunagaraj** [comment]
@anisha147 Microsite is not updated with the R5.0.1 Tickets which have been signed off for Sunbird Ed. Kindly update the Microsite.

CC @luckynvdu @sharathkashyap @ShailandraOjha @chitranshu-keshav

**Reply by anisha147** [reply]
@anunagaraj Release notes have been added to the microsite. 

cc: @luckynvdu @sharathkashyap @ShailandraOjha @chitranshu-keshav
